crates/router/src/compatibility/stripe/payment_intents/types.rs

```diff
@@ -340,10 +350,19 @@ impl From<payments::PaymentsResponse> for StripePaymentIntentResponse {
             statement_descriptor_suffix: resp.statement_descriptor_suffix,
             next_action: into_stripe_next_action(resp.next_action, resp.return_url),
             cancellation_reason: resp.cancellation_reason,
-            error_code: resp.error_code,
-            error_message: resp.error_message,
             metadata: resp.metadata,
             charges: Charges::new(),
+            last_payment_error: resp.error_code.map(|code| LastPaymentError {
+                charge: None,
+                code: Some(code.to_owned()),
+                decline_code: None,
+                message: resp
+                    .error_message
+                    .unwrap_or_else(|| "Error message is null".to_string()),
```

Comment:
Is this what stripe sends if there is no error message?

Reply:
No, Stripe either sends the error message or not. But in this case we have to handle the optional value. So used the above message as placeholder.

Reply:
in that case, use consts::NO_ERROR_MESSAGE